Package.swift

```diff
@@ -17,13 +17,12 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(url: "https://github.com/GetStream/stream-chat-swift.git", from: "4.39.0"),        
-        .package(url: "https://github.com/kean/Nuke.git", .exact("11.3.1"))
+        .package(url: "https://github.com/GetStream/stream-chat-swift.git", from: "4.39.0"),
     ],
     targets: [
         .target(
             name: "StreamChatSwiftUI",
-            dependencies: [.product(name: "StreamChat", package: "stream-chat-swift"), "Nuke", .product(name: "NukeUI", package: "Nuke")],
```

Comment:
I still see Difference, Swifter and the tools. Can we also get rid of those? (not in this PR)

Reply:
Those come from StreamChat. They will be gone as soon as https://github.com/GetStream/stream-chat-swiftui/pull/376 is merged